### DISCUSSION TITLE
Skinning in osu!lazer

### DISCUSSION THREAD

**Discussion by Rebta** [ppy/osu#16226] [Ideas] [answered]
Add a skin template in osu!lazer to help new osu!skinner (and to know everything that we can custom).

**Comment by bdach** [comment, marked as answer]
the system from stable is considered legacy and a new one will be eventually coming, so there will be no progress on this until work on that has started. for help on the legacy/stable format please read the [wiki](https://osu.ppy.sh/help/wiki/Skinning/)
